bitnami/jaeger/templates/agent/deployment.yaml

```diff
@@ -0,0 +1,184 @@
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name:  {{ include "jaeger.agent.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+  replicas: {{ .Values.agent.replicaCount }}
+  {{- if .Values.agent.updateStrategy }}
+  strategy: {{- toYaml .Values.agent.updateStrategy | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels: {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: query
```

Comment:
The component should be `agent` here, right? Review the labels in all the yaml

Reply:
Still not done

Reply:
Done [7cd2e8a](https://github.com/bitnami/charts/pull/13480/commits/7cd2e8aa1ffceca40db117a274fab7a4d4102e18). Also added `app.kubernetes.io/part-of: jaeger` where missing